src/batch-client/index.ts

```diff
@@ -162,20 +174,30 @@ export class ZimbraBatchClient {
 	public conversationAction = (options: ActionOptions) =>
 		this.action(ActionType.conversation, options);
 
-	public createAppointment = (appointment: CalendarItemInput) =>
+	public createAppointment = (
+		accountName: string,
+		appointment: CalendarItemInput
+	) =>
 		this.jsonRequest({
 			name: 'CreateAppointment',
 			body: {
 				...denormalize(CalendarItemCreateModifyRequest)(appointment)
-			}
+			},
+			accountName: accountName,
+			namespace: Namespace.Mail
```

Comment:
It looks like the namespace will default to `Namespace.Mail`, so would it be OK to remove this extra option?

Reply:
Done!